ocaml/typing/typedecl.ml

```diff
@@ -2680,9 +2681,11 @@ let make_native_repr env core_type ty ~global_repr ~is_layout_poly ~why =
        - this isn't a tvar from an outer scopes ([TyVarEnv] gets reset before
          transl)
     *)
+    (* FIXME jbachurski: Is this the way this unwrapping should happen?
```

Comment:
Say more? The code near here does not seem to be edited, so I'm not sure what this comment is really about.

Reply:
Accidental - it says `Tvar { jkind = Type jkind }` in the implementation, but I later found this was okay, as this pattern match only occurs in a context where this has to be a type jkind.